src/ex5/server/routes/api.js

```diff
@@ -1,19 +1,19 @@
-const express = require('express');
-const router = express.Router();
-const itemManager = require('../services/item_manager');
+const express = require("express")
+const router = express.Router()
+const itemManager = require("../services/item_manager")
 
-router.get('/items', (_, res) => {
-    res.send(itemManager.getItems())
+router.get("/items", async (_, res) => {
+  res.send(await itemManager.getItems())
 })
 
-router.post('/item', async (req, res) => {
-    await itemManager.handleItem(req.body.item)
-    res.end()
+router.post("/item", async (req, res) => {
+  await itemManager.handleItem(req.body.item)
```

Comment:
the router should just mention the controller action, the controller should take care of body and params seperation